lib/std/math.zig

```diff
@@ -6,6 +6,17 @@ const testing = std.testing;
 
 pub const e = 2.71828182845904523536028747135266249775724709369995;
 pub const pi = 3.14159265358979323846264338327950288419716939937510;
+pub const log2e = 1.4426950408889634074; // log2(e)
```

Comment:
Are these to sufficient precision for e.g. an `f128`?

Reply:
I've updated the consts to more precise values.  We can not calculate those values within zig (yet) as log is not implemented for f128.